### PR TITLE
Replace 'CASA/Prince George' with User casa_org name

### LINKED ISSUE #931: replace org specific language from system emails with `casa_org`
AS a volunteer/supervisor/admin,
I want to see my `casa_org` referenced if at all in system emails,
so that I know the email is intended for me.

**How does it work now?**
There are likely a number of system emails that reference "PG / CASA", regardless of what `casa_org` a user is associated with.

**How should it work instead?**
System emails should only reference the `casa_org` associated with the recipient user.

**Acceptance Criteria**
When a user receives a system email,
if any `casa_org` is referenced, it should be the one they belong to.

### PR DESCRIPTION
**Pull request by Garbar**
### What github issue is this PR for, if any?
Resolves #931 

### What changed, and why?
Replace 'CASA/Prince George' with User casa_org name in emails from the site
For Devise::Mailer added method for users

### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### CODE REVIEW

File: app/mailers/volunteer_mailer.rb [hunk 1]
```diff
@@ -3,11 +3,13 @@ class VolunteerMailer < ApplicationMailer
 
   def deactivation(user)
     @user = user
+    @casa_organization = CasaOrg.find(@user.casa_org_id)
```
Comment (jmkoni) [inline app/mailers/volunteer_mailer.rb:6]: Do we need this? Why not just use @user.casa_org in the view?

File: app/models/user.rb [hunk 2]
```diff
@@ -128,6 +128,10 @@ def inactive_message
   def serving_transition_aged_youth?
     casa_cases.where(transition_aged_youth: true).any?
   end
+
+  def casa_org_display_name
```
Comment (jmkoni) [inline app/models/user.rb:132]: This can just be `casa_org.display_name`

**Review by jmkoni — APPROVED** [review]
Looks good!

**Comment by jmkoni** [comment]
Question for @Zrrrpy or @compwron: would we want to have a default value if this isn’t set?

**Comment by compwron** [comment]
> Question for @Zrrrpy or @compwron: would we want to have a default value if this isn’t set?

This should always be set. For now, a default value seems unneeded.

**Review by compwron — APPROVED** [review]
Very nice :) 🙌🏼

**Pull request merged**